packages/host/app/components/operator-mode/stack-item.gts

```diff
@@ -69,20 +70,22 @@ export default class OperatorModeStackItem extends Component<Signature> {
     fieldType: FieldType | undefined;
   }>();
 
-  get renderedLinksToCards(): RenderedLinksToCard[] {
+  get renderedCardsForOverlayActions(): RenderedCardForOverlayActions[] {
     return (
       this.cardTracker.elements
         .filter((entry) => {
           return (
             entry.meta.format === 'data' ||
             entry.meta.fieldType === 'linksTo' ||
-            entry.meta.fieldType === 'linksToMany'
+            entry.meta.fieldType === 'linksToMany' ||
+            entry.meta.fieldType === 'contains'
           );
         })
         // this mapping could probably be eliminated or simplified if we refactor OperatorModeOverlays to accept our type
         .map((entry) => ({
           element: entry.element,
           card: entry.meta.card,
+          fieldType: entry.meta.fieldType,
```

Comment:
This is to know whether we need to render the overlaid header or not. Embedded cards (links to, contains) will have the header